test/integration/teacher-registration/test_teacher_registration_address_step.js

```diff
@@ -0,0 +1,80 @@
+/*
+ * Checks the behavior of the address step in the educators registration process
+ *
+ * Test cases: https://github.com/LLK/scratch-www/wiki/Testing-Scratch-www#All_Test_Cases_Teacher_Join_Flow
+ */
+require('chromedriver');
+var seleniumWebdriver = require('selenium-webdriver');
+var tap = require('tap');
+
+var utils = require('./teacher_registration_utils.js');
+var constants = utils.constants;
+
+//Set test url through environment variable
+//var rootUrl = process.ENV.ROOT_URL || 'http://localhost:8333';
+
+//chrome driver
+var driver = new seleniumWebdriver.Builder().withCapabilities(seleniumWebdriver.Capabilities.chrome()).build();
+
+var fillUsernameSlide = function () {
+    return utils.fillUsernameSlide(driver, seleniumWebdriver);
+};
+
+var fillDemographicsSlide = function () {
+    return utils.fillDemographicsSlide(driver, seleniumWebdriver);
+};
+
+var fillNameSlide = function () {
+    return utils.fillNameSlide(driver, seleniumWebdriver);
+};
+
+var fillPhoneSlide = function () {
+    return utils.fillPhoneSlide(driver, seleniumWebdriver);
+};
+
+var fillOrganizationSlide = function () {
+    return utils.fillOrganizationSlide(driver, seleniumWebdriver);
+};
+
+tap.plan(2);
+
+tap.tearDown(function () {
+    driver.quit();
+});
+
+tap.beforeEach(function () {
+    driver.get('https://scratch.mit.edu/educators/register');
```

Comment:
Tracking for later – make `https://scratch.mit.edu` into a `process.ENV` variable that defaults to `http://localhost:8333`